manager/Makefile

```diff
@@ -57,6 +59,7 @@ wait_for_manager: $(TOOLBIN)/kubectl
 .PHONY: run-integration-tests
 run-integration-tests: export DOCKER_HOSTNAME?=localhost:5000
 run-integration-tests: export DOCKER_NAMESPACE?=fybrik-system
+run-integration-tests: export USE_WKC_CONTROLLER?=true
```

Comment:
Can USE_EXISTING_CLUSTER be reused instead? Running "mockup" tests in the real environment will fail in any case.

Reply:
If I'm not mistaken, these particular tests will run successfully on an existing cluster.

Reply:
On an existing cluster, with mockup connectors - yes. Perhaps I misunderstood the purpose of this controller. 
Anyway, my intent was to avoid any wkc related content appearing in the open-source repo.